news/_posts/2019-05-23-announcing-end-of-support-for-obsolete-php-versions.md

```diff
@@ -0,0 +1,64 @@
+
+
+## PrestaShop is directly affected by this change. But how exactly?
+
+First and foremost, as you probably know, the internet is a dangerous place... and businesses are prime targets for attackers. As a merchant or shop maintainer, you need to be aware that it's dangerous to run any web service on outdated software, whether it is your operating system, web server, PrestaShop itself or the scripting language it's written in, PHP. Outdated software may contain a known vulnerability that could be exploited by an attacker to take over your shop and do serious damage to your business. Keeping your software stack up to date ensures you get the latest security fixes, and reduces the risk that your shop will be compromised.
+
+PrestaShop has many dependencies, PHP being just one of them. Dependency management has two sides: the minimum supported version, and the maximum supported version. In our case, PrestaShop 1.7.6 supports all versions of PHP from 5.6 to 7.2, but not the latest 7.3. Why not? Some of the [breaking changes](https://www.php.net/manual/en/migration73.incompatible.php) introduced in that version [affect libraries that PrestaShop depends on](https://github.com/PrestaShop/PrestaShop/issues/12461) and has no control over, in particular [Doctrine](https://github.com/doctrine/orm/issues/7402). In that specific case, the version of Doctrine where the fix was applied has dropped support for PHP 5, which puts us in the difficult situation of having to choose to either to support older PHP versions or support newer ones.
+
+Doctrine is not the only library to stop supporting older PHP versions. Many other libraries like PHP-Parser and PhpUnit have dropped support for PHP 5 for a long time. Even some quality tools, like PhpStan, never supported it in the first place, preventing us from using them.
+
+In the case of Symfony, while they dropped support for PHP 5 on Symfony 4, PrestaShop is based on the 3.4 long-term-support (LTS) version, which is unlikely to drop support for PHP 5 in the near future.
+
+
+![Symfony version support roadmap](/assets/images/2019/05/symfony-support-roadmap.png){:.text-center}
+
+_Symfony version support roadmap. ([Source](https://symfony.com/roadmap#maintained-symfony-branches))_
+{:.text-center}
+
+Even though Symfony is the exception, the PHP dependency problem is becoming an increasingly complex problem, and we had to make a choice. Many other PHP-based CMS are facing the same problems: [Drupal dropped compatibility with PHP 5 back in March](https://www.drupal.org/node/2938726), while Wordpress, who has traditionally supported very old PHP versions, is now [requiring at least 5.6 and will soon drop it as well](https://wordpress.org/news/2019/04/minimum-php-version-update/).
+
+
+## Here is what we decided
```

Comment:
```suggestion
## Here is what has been decided
```
on Build, we must avoid the "we", not clear, giving the impression that the PrestaShop is the only contributor to the project, and try to be more inclusive. Plus, many advice from people outside the company have been taken in account.